eras/shelley-ma/impl/src/Cardano/Ledger/Mary/Translation.hs

```diff
@@ -153,7 +153,7 @@ instance Crypto c => TranslateEra (MaryEra c) ShelleyTxOut where
 
 instance Crypto c => TranslateEra (MaryEra c) UTxO where
   translateEra ctxt utxo =
-    return $ UTxO (translateEra' ctxt <$> unUTxO utxo)
+    return $ UTxO (translateEra' ctxt `Map.map` unUTxO utxo)
```

Comment:
`Map` has `Functor` instance, why this change?

Reply:
This change was part of the original PR. Will check the implementation and then revert this change if it is unnecessary :+1:

Reply:
`fmap` is indeed defined in terms of `map`, for both `v0.6.6` as well as `0.6.5.1` of `containers`. Reverting this change. :+1:

Reply:
@nfrisby might clarify, but I think he did this to prevent extra thunks as the `Functor Map` instance is lazy in the values, while `Data.Map.Strict.map` is strict.

Reply:
@amesgen That is very good point. We probably want to add a comment about this, so no-one switches it back to `fmap` in the future.

Reply:
@amesgen @nfrisby sorry didn't catch that while I was looking!

Reply:
Here is a separate PR: https://github.com/input-output-hk/cardano-ledger/pull/3143

Reply:
Yes, that diff should only be in PR 3143, not this one. It was merely a "leftover" in my commits.